pvt/helpers/src/models/pools/weighted/WeightedPoolDeployer.ts

```diff
@@ -122,6 +124,32 @@ export default {
         });
         break;
       }
+      case WeightedPoolType.MOCK_MANAGED_POOL: {
+        assert(mockContractName != undefined, 'Mock contract name required to deploy mock base pool');
```

Comment:
```suggestion
        if (mockContractName == undefined) {
           throw new Error('Mock contract name required to deploy mock base pool');
        }
```

Reply:
Thanks; I was in doubt because I only saw it in benchmarks :)